app/src/main/java/fr/free/nrw/commons/contributions/ContributionViewHolder.java

```diff
@@ -229,35 +241,30 @@ private String chooseImageSource(final String thumbUrl, final Uri localUri) {
     /**
      * Retry upload when it is failed
      */
-    @OnClick(R.id.retryButton)
     public void retryUpload() {
         callback.retryUpload(contribution);
     }
```

Comment:
If this is only called from one place (the click listener) then you could probably inline and delete the method.

Reply:
These methods ae also used in ContributionViewHolderUnitTests.kt, that's why I kept them

<img width="498" alt="Screenshot 2024-03-12 at 10 35 51 AM" src="https://github.com/commons-app/apps-android-commons/assets/126143257/dbc80b69-f174-4bcf-9ff7-a0b42da1ba54">